### PR TITLE
Added UMC operations

### PR DESCRIPTION
**Pull request by assing**

### CODE REVIEW

File: specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json [hunk 1]
```diff
@@ -1858,6 +1993,465 @@
         }
       },
       "description": "Describes the subnet."
+    },
```
Comment (edyoung) [inline specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json:1996]: This change doesn't seem to be related to a UMC operation, and is where all the breaking change notifications are coming from.

File: specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json [hunk 2]
```diff
@@ -1858,6 +1993,465 @@
...
+    "ErrorDetail": {
```
Comment (edyoung) [inline specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json:1997]: I *think* the breaking change notifications are happening because there is already an ErrorDetails defined as a common type, and the linter thinks (correctly or incorectly) that you're overriding it here. 

I think if you remove these and instead reference the common ErrorDetails and ErrorResponse types (search the json for ErrorDetails for examples) it will be happier, and because the issues are about new required properties it won't be an issue because these are only output types, not things the user submits.

Comment (assing) [reply specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json]: Done
